index.html

```diff
@@ -49,5 +49,167 @@ <h1><span class="hero__title-dark">Hey there. I'm Daniel Selase</span><span clas
             </div>
         </section>
     </header>
+
+    <!-- Recent Works -->
+    <section class="work">
+        <div class="container">
+            <div class="work__heading">
+                <h2 class="heading-2"><span class="work__heading-recent">My Recent</span> <span class="work__heading-work">Works</span></h2>
+            </div>
+            <div class="projects">
```

Comment:
:red_circle:  Great implementation of the grid layout :clap: :1st_place_medal: as requested by the previous reviewer. But  I think you applied it to the wrong element  ` <ul class="card__tags-works">`. The [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) states that this layout should be applied to the parent container. Please read the second line in the screenshot :point_down: 

![requirement](https://user-images.githubusercontent.com/26736582/200043935-074b671b-0b02-48b3-90f4-4e73a7124d9d.png)

I think according to the project requirement your Grid layouts should be applied to this element with the class name projects  which is this :arrow_right: ` <div class="projects">`